docs/4.4/admin-guide.md

```diff
@@ -1458,6 +1458,16 @@ teleport:
      # NOT RECOMMENDED: enables insecure etcd mode in which self-signed
      # certificate will be accepted
      insecure: false
+
+     # Optionally sets the limit on the client message size.
+     # This is usually used to increase the default which is 2MiB
+     # (1.5MiB server's default + gRPC overhead bytes).
+     # Make sure this does not exceed the value for the etcd
+     # server specified with `--max-request-bytes` (1.5MiB by default).
+     # Keep the two values in sync.
+     #
+     # See https://etcd.io/docs/v3.4.0/dev-guide/limit/ for details
+     max_client_msg_size_bytes: 15728640 # 15MiB
```

Comment:
Given that this setting is etcd-specific and this is a section for all backends, I'd add "etcd" somewhere in the name.

Also, do you think it actually needs to be exposed, rather than just setting a reasonably large default, like 10MB?

Reply:
Setting a higher default might result in undesirable performance costs but I don't have the data to back it up. But I did want to provide a configuration knob as a last resort since there seems to be no better way to work around a failing set (other than consciously splitting the large values into smaller chunks which I'm not sure is feasible).